projects/vdk-core/src/vdk/api/plugin/plugin_input.py

```diff
@@ -167,7 +167,6 @@ def initialize_job(context: JobContext) -> None:
 class IIngesterPlugin:
```

Comment:
Can you write pydoc here explaining the the lifecycle of the 3 methods .

Reply:
Done, thanks